subjects/printandmiss/README.md

```diff
@@ -0,0 +1,47 @@
+# printandmiss
+
+### Instructions
+
+Write a function called `PrintAndMiss` that takes a string and an integer and prints a string containing the characters until reaching the integer, then skipping that same number of characters, and repeats until the end of the string.
+
+- Prints the first half followed by newline `\n`.
+- If the string is empty retun `Invalid Output`.
```

Comment:
Add `followed by a new line`  or remove `$` in `Invalid Output` in the usage part